charts/kube-prometheus-stack/values.yaml

```diff
@@ -2712,7 +2712,7 @@ prometheusOperator:
   thanosImage:
     registry: quay.io
     repository: thanos/thanos
-    tag: v0.34.0
+    tag: v0.34.1
```

Comment:
This file come from the helm chart so it shouldn't be updated, you need to update the full charts using the `helm` commands

Basically we do not update by hand any files in `charts/<name>/*` (just the options file so `charts/<name>.yaml`)

Reply:
Yes, it's only a test to check if CI failed as my newly devcontainer with go.
I've supposed that my changes with kube-prometheus-stack breaks everyhing :/
You said that you fixed that in other commit, so I've rebased and will push it